docs/adr/0007-wikibase-release-notes-publish.md

```diff
@@ -0,0 +1,27 @@
+# 5) Wikibase Extension release notes publishing {#adr_0007}
+
+Date: 2021-01-14
+
+## Status
+
+proposed
+
+## Context
+
+We have in [ADR0005] adopted the process for maintaining our release notes within the repository we also want to make a decision as to where these should be published for a broader audience.
+
+As our process for maintaining release notes will be closely modeled after mediawiki it also makes sense to review how mediawiki is publishing theirs.
+
+The [template] used for describing the different release notes topics is based on wikitext that after a release branch is cut and the template has been populated with the relevant changes, is to serve as the base for a richer curated version on mediawiki.org [(Example: 1.35)](https://www.mediawiki.org/wiki/Release_notes/1.35).
```

Comment:
"that after a release branch is cut" I think this could be clarified a little.

Reply:
I updated this section a bit but still not going into too much detail on how the branches are cut, is this sufficient?